oauth2_http/java/com/google/auth/oauth2/S2AConfig.java

```diff
@@ -0,0 +1,58 @@
+package com.google.auth.oauth2;
+
+import com.google.errorprone.annotations.CanIgnoreReturnValue;
+
+/** Holds an mTLS configuration (consists of address of S2A) retrieved from the Metadata Server. */
+public final class S2AConfig {
```

Comment:
Does this need to be public?

Reply:
It does not, thanks for pointing that out. Done in https://github.com/googleapis/google-auth-library-java/pull/1400/commits/36a0ac7cad28656f791a057bdcac70bb6cfd1e27